src/components/core/core.js

```diff
@@ -346,6 +346,7 @@ export default class Core extends UIObject {
     this.containers.forEach((container) => {
       container.configure(this.options)
     })
+    this.mediaControl.configure(this.options)
```

Comment:
Would be nice to check if  `configure` is a method, because some Clappr use cases can add an external Media Control without this new configure method, and will get an 
Uncaught TypeError: `mediaControl.configure is not a function`

Reply:
Shouldn't all the (configurable) components answer to this method? (even if it's to be an empty implementation)

Reply:
Once we define that a refresh on any configuration will trigger a calling to all configurable components with the newConfig, this might be helpful.

Reply:
@vagnervjs looks like I should follow [this style](https://github.com/clappr/clappr/blob/147867b33c9bcd1268aff6505bbbdb33d2a01e62/src/base/polyfills.js#L27). Should I open a new PR with these changes?